memdx/ops_crud.go

```diff
@@ -166,7 +166,8 @@ type GetResponse struct {
 }
 
 func (o OpsCrud) Get(d Dispatcher, req *GetRequest, cb func(*GetResponse, error)) (PendingOp, error) {
-	reqMagic, extFramesBuf, err := o.encodeReqExtFrames(req.OnBehalfOf, 0, 0, false, nil)
+	extFramesBuf := make([]byte, 0, 128)
```

Comment:
What's the reason for declaring this and passing it into `encodeReqExtFrames` vs just passing in nil as was done before?

Reply:
At a guess it preallocates the capacity so it doesn't need to grow the array?

Reply:
Go has an optimization where if you allocate an array with a constant size (up to some limit) and the compiler is able to prove that the allocated memory does not need to escape from the heap, it actually allocates on the heap instead which is what occurs here.  This `make([]byte, 0, 128)` call doesn't actually perform an allocation.

Reply:
I see, thanks Brett